app/models/applicant_state.rb

```diff
@@ -36,26 +35,36 @@ def ttl_paid
     @current_enrollment.user.payments.current_camp_payments.where(transaction_status: '1').pluck(:total_amount).map(&:to_i).sum
   end
```

Comment:
status1_current_camp_payments scope might be used in line 35

Reply:
Not necessary and adds unneeded DB calls